.gitignore

```diff
@@ -1,4 +1,4 @@
-node_modules
+smart-contracts/node_modules
```

Comment:
Hi, hätte das so gelassen, weil dann alle Node Modules in allen Ordner ignoriert werden :D

Reply:
Stimmt